src/gameplay/cards/MonsterCard.js

```diff
@@ -77,23 +77,54 @@ class MonsterCard extends Card {
   }
 
   takeSnapshot() {
+    let summonAction;
+
+    if (this.lv < 5) {
+      summonAction = {
+        name: 'summon',
+        desc: 'summon this monster to the field',
+        in: Game.HAND,
+        params: [
+          {select: Game.SLOT, in: Game.MONSTER_SLOTS, of: Game.SELF, desc: 'select an empty slot in the monster slots to summon'},
+          {select: Game.DISPLAY, desc: 'select the display of the monster'},
+          {select: Game.POSE, desc: 'select the pose of the monster'},
+        ],
+      };
+    } else if (this.lv < 7) {
+      summonAction = {
+        name: 'summonTribute1',
+        desc: 'summon this monster to the field with one tribute',
+        in: Game.HAND,
+        params: [
+          {select: Game.CARD, in: Game.MONSTER_SLOTS, of: Game.SELF, desc: 'select a monster to tribute'},
+          {select: Game.SLOT, in: Game.MONSTER_SLOTS, of: Game.SELF, desc: 'select an empty slot in the monster slots to summon'},
+          {select: Game.DISPLAY, desc: 'select the display of the monster'},
+          {select: Game.POSE, desc: 'select the pose of the monster'},
+        ],
+      };
+    } else {
+      summonAction = {
+        name: 'summonTribute2',
+        desc: 'summon this monster to the field with two tribute',
+        in: Game.HAND,
+        params: [
+          {select: Game.CARD, in: Game.MONSTER_SLOTS, of: Game.SELF, desc: 'select the first monster to tribute'},
+          {select: Game.CARD, in: Game.MONSTER_SLOTS, of: Game.SELF, desc: 'select the second monster to tribute'},
+          {select: Game.SLOT, in: Game.MONSTER_SLOTS, of: Game.SELF, desc: 'select an empty slot in the monster slots to summon'},
```

Comment:
We do not limit the character count for one line since we believe that long lines are easier to read under some circumstances.

Please see our Eslint config:
https://github.com/DotLab/touhou-card-game-nodejs/blob/98569c8bc43dc3c211a42dee737da77b954faffe/.eslintrc.json#L14

The refactor did make the line shorter and easier to read nevertheless.